tests/nodes/NodeConnectionManager.seednodes.test.ts

```diff
@@ -453,8 +453,8 @@ describe(`${NodeConnectionManager.name} seed nodes test`, () => {
       await nodeConnectionManager.start({ nodeManager });
       await taskManager.startProcessing();
       // This should complete without error
-      await nodeManager.syncNodeGraph(true, 5000, {
-        timer: new Timer({ delay: 20000 }),
+      await nodeManager.syncNodeGraph(true, 2000, {
+        timer: new Timer({ delay: 15000 }),
       });
```

Comment:
What do these constants here mean?

Reply:
2000 is the time for each individual connection, 15000 is for the overall operation.

Reply:
Should this be taken from overall app configuration? Or is there something special about this test that requires special cased times?

Reply:
What about our composed timer spec? Should this be updated so that the individual timeouts can be auto-derived?

Reply:
Just custom parameters for the test here, I'm using a 15 second overall timeout so we get a better error than the jest timeout.